tests/platform_tests/api/test_thermal.py

```diff
@@ -83,6 +83,17 @@ def get_thermal_facts(self, duthost, thermal_idx, def_value, *keys):
 
         return def_value
 
+    def get_thermal_temperature(self, duthost, def_value, *keys):
+        if duthost.facts.get("chassis"):
+            thermals_temperature = duthost.facts.get("chassis").get("thermal_temperature")
+            if thermals_temperature:
+                for key in keys:
+                    value = thermals_temperature.get(key)
+                    if value is None:
+                        return def_value
+                return value
```

Comment:
Only `value` of the last key in `keys` will be returned if all the keys are defined in the `thermals_temperature` dict. It's meaningless to support passing in multiple keys in argument.
Suggest to pass in single key argument:
```
def get_thermal_temperature(self, duthost, def_value, key):
```